libparsec/crates/types/src/id.rs

```diff
@@ -675,15 +675,26 @@ pub enum UserProfile {
     Outsider,
 }
 
+#[derive(Debug, Clone)]
+pub struct InvalidUserProfile(String);
```

Comment:
```suggestion
pub struct InvalidUserProfile;
```

No need to display the invalid data

Reply:
done